runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkPipelineOptions.java

```diff
@@ -184,4 +186,14 @@
   Long getLatencyTrackingInterval();
 
   void setLatencyTrackingInterval(Long interval);
+
+  @Description(
+      "SDK process parallelism for portable pipelines. Currently supported options are "
+          + "'[auto]' (Let the runner decide) or "
+          + "'[pipeline]' (single SDK harness process per pipeline and task manager JVM) or "
+          + "'[stage]' (separate SDK harness for every executable stage.)")
+  @Default.String(AUTO)
+  String getSdkWorkerParallelism();
```

Comment:
I think this change is great but I wonder whether we should expose this as a general PortablePipelineOptions?

Reply:
Yes, done!